packages/rpc-methods/src/restricted/invokeSnap.ts

```diff
@@ -181,7 +181,7 @@ export function getInvokeSnapImplementation({
 
     if (!getSnap(snapId)) {
       throw ethErrors.rpc.invalidRequest({
-        message: `The snap "${snapId}" is not installed. This is a bug, please report it.`,
+        message: `The snap "${snapId}" is not installed. Please install it first, before invoking the snap.`,
```

Comment:
Now that I think about it, this isn't a bug _right now_. But once we have side effects it will be? Since the snap should always be installed if you have the permission?

Reply:
And we need better snap dependency handling, but thats for v2. Requesting a snap will be a wonky edge-case in v1